src/app/my-groups/group-details/group-details.page.ts

```diff
@@ -385,16 +385,16 @@ export class GroupDetailsPage implements OnInit, OnDestroy, ViewMoreActivityActi
         return;
       }
 
-      // this.telemetryGeneratorService.generateInteractTelemetry(
-      //   InteractType.INITIATED,
-      //   '',
-      //   Environment.GROUP,
-      //   PageId.GROUP_DETAIL,
-      //   undefined,
-      //   undefined,
-      //   undefined,
-      //   this.corRelationList,
-      //   ID.DELETE_GROUP);
+      this.telemetryGeneratorService.generateInteractTelemetry(
+        InteractType.INITIATED,
+        '',
+        Environment.GROUP,
+        PageId.GROUP_DETAIL,
+        undefined,
+        undefined,
+        undefined,
+        this.corRelationList,
+        ID.DEACTIVATE_GROUP);
```

Comment:
@balakrishna10  Create a method that will accept InteractType and ID and reuse it across the page.